lib/octokit/preview.rb

```diff
@@ -13,7 +13,7 @@ module Preview
       :source_imports         => 'application/vnd.github.barred-rock-preview'.freeze,
       :reactions              => 'application/vnd.github.squirrel-girl-preview'.freeze,
       :transfer_repository    => 'application/vnd.github.nightshade-preview+json'.freeze,
-      :issue_timelines        => 'application/vnd.github.mockingbird-preview+json'.freeze,
+      :timeline_events        => 'application/vnd.github.mockingbird-preview+json'.freeze,
```

Comment:
Was this auto generated?

Reply:
No this wasn't 😬I changed it to be consisted with the preview that was autogenerated in the code. I double checked I wasn't breaking anything and I was having a really hard time autogenerating `issue_timelines`.